pkgs/top-level/all-packages.nix

```diff
@@ -8761,6 +8761,8 @@ with pkgs;
     inherit (perlPackages) IOStringy;
   };
 
+  hl1110 = callPackage_i686 ../misc/cups/drivers/hl1110 { };
```

Comment:
It belongs near other `cups/drivers` in `all-packages.nix`.

Could you rename it to `cups-brother-hl1110`?

Reply:
Just wanted to let you know that I've read your comment, but I haven't been in the position to test the change; even though it is trivial. Will get to it this week.